packages/tbas/turbo.json

```
@@ -0,0 +1,15 @@
+{
+  "extends": ["//"],
+  "globalEnv": [
+    "TURBO_ENV",
+    "XDC_WALLET_SECRET"
+  ],
+  "tasks": {
+    "build": {
+      "dependsOn": [
+        "@cfce/blockchain-tools#build",
+        "@cfce/types#build"
+      ]
+    }
+  }
```

Build automatically depend on their dependent packages
```suggestion
  ]
```